src/playlist-controller.js

```diff
@@ -929,6 +929,31 @@ export class PlaylistController extends videojs.EventTarget {
       this.onEndOfStream();
     });
 
+    // In DASH, there is the possibility of the video segment and the audio segment
+    // at a current time to be on different timelines. When this occurs, the player
+    // forwards playback to a point where these two segment types are back on the same
+    // timeline. This time will be just after the end of the audio segment that is on
+    // a previous timeline.
+    this.timelineChangeController_.on('audioTimelineBehind', () => {
```

Comment:
sounds like we can listen to this event only when dash

Reply:
I guess leaving that comment was to point out that this scenario should only happen when DASH.. but I am not sure if it hurts to handle it in any case.

I can probably make that more clear

Reply:
I mean, if this code is explicitly intended to handle a dash-specific case, why should we run it for HLS (and potentially impact HLS playback)

I think we should guard it with if statement